atom/browser/web_contents_permission_helper.cc

```diff
@@ -102,4 +113,14 @@ void WebContentsPermissionHelper::RequestOpenExternalPermission(
       callback, user_gesture, &details);
 }
 
+bool WebContentsPermissionHelper::CheckMediaAccessPermission(
+    const GURL& security_origin,
+    content::MediaStreamType type) {
+  base::DictionaryValue details;
+  details.SetString("securityOrigin", security_origin.spec());
+  // The permission type doesn't matter here, AUDIO_CAPTURE/VIDEO_CAPTURE
+  // are presented as same type in content_converter.h.
```

Comment:
Perhaps we should fix that? It seems relevant to know whether the webcontents is requesting audio or video.

Reply:
Currently they both get converted to `media`, I agree that the user should be able to tell the different between all the `content::MediaStreamType`'s.  We could just set the media stream type on the details object 🤔 

I did it this way to be consistent with the existing media permission stuff in `setPermissionRequestHandler`

Reply:
ah right, well we should fix that, but perhaps in a separate PR :)